### PR TITLE
ADBDEV-4740: Fix backup table ordering tests

### PR DESCRIPTION
**Pull request by RekGRpth**
Fix backup table ordering tests

Upstream commit 7833212 removed the ExpandIncludesForPartitions function,
but it continued to be used in our tests in commit 1ddc17e. Because of this,
integration tests could not run and would throw compilation errors.
```
Failed to compile integration:
./wrappers_test.go:168:16: opts.ExpandIncludesForPartitions undefined (type *options.Options has no field or method ExpandIncludesForPartitions)
./wrappers_test.go:184:16: opts.ExpandIncludesForPartitions undefined (type *options.Options has no field or method ExpandIncludesForPartitions)
```
I corrected our tests so that there were no compilation errors
and they passed successfully.

### CODE REVIEW

**Review by dnskvlnk — APPROVED** [review]

File: integration/wrappers_test.go [hunk 1]
```diff
@@ -18,12 +18,12 @@ var _ = Describe("Wrappers Integration", func() {
...
 		BeforeEach(func() {
 			rootCmd := &cobra.Command{}
-			includes := []string{"--include-table", "public.foo", "--include-table", "public.BAR"}
-			rootCmd.SetArgs(options.HandleSingleDashes(includes))
 			backup.DoInit(rootCmd) // initialize the ObjectCount
-			rootCmd.Execute()
 		})
 		It("returns the data tables that have names with special characters", func() {
+			_ = backupCmdFlags.Set(options.INCLUDE_RELATION, "public.foo")
+			_ = backupCmdFlags.Set(options.INCLUDE_RELATION, "public.BAR")
```
Comment (red1452) [inline integration/wrappers_test.go:25]: Why did you move it?

File: integration/wrappers_test.go [hunk 2]
```diff
@@ -164,8 +193,12 @@ var _ = Describe("Wrappers Integration", func() {
 			rootCmd := &cobra.Command{}
 			backup.DoInit(rootCmd) // initialize the ObjectCount
 
-			opts, _ := options.NewOptions(rootCmd.Flags())
-			err := opts.ExpandIncludesForPartitions(connectionPool, rootCmd.Flags())
```
Comment (red1452) [inline integration/wrappers_test.go:168]: At the upstream [commit](https://github.com/arenadata/gpbackup/commit/783321238283de51dd3bb9c771e450643047e052) I see, that this function was changed (two new arguments was added). Can we do the same?

Comment (RekGRpth) [reply integration/wrappers_test.go:168]: > Can we do the same?

[Yes](https://github.com/arenadata/gpbackup/pull/51/files#diff-afc5c2c573c4cffee815db38bae03c14ed1bf8ca90bc68b99ccaa6d5f4b0e418R201)

File: integration/wrappers_test.go (hunk 1 above)
Comment (RekGRpth) [reply integration/wrappers_test.go:25]: This is a partial revert of Polina's changes to be closer to the upstream.

**Review by red1452 — APPROVED** [review]
